### DISCUSSION TITLE
Undocumented/untested attributes

### DISCUSSION THREAD

**Discussion by radare** [vlang/v#11658] [Ideas]
**V version:** V 0.1.25 c12985d
**OS:** macOS 10.15

**What did you do?**

grepping in the source i found all those `[..]` attributes used or documented. its a bit confusing to understand what are they for and how/when to use them. so it will be good to have some tests and document them in doc/attributes.md

Also it is possible to define custom attributes like in python or c# ?

```
typedef
live
flag
inline
unsafe_fn
deprecated
testing
if vfmt
skip
```

**What did you expect to see?**

 
**What did you see instead?**

**Comment by nedpals** [comment]
For documenting the attributes, this should be filed inside the [vlang/website](https://github.com/vlang/website) repo.

And please don't tag this issue as a `bug`.
